src/util/generate_grid_nodes.js

```diff
@@ -15,4 +15,18 @@ const generateGridNode = (rowCount, colCount) => {
   return gridNodes;
 };
 
+/**
+ * Create new gridCell as above but now highlighted by default
+ */
+export const generateFilledGrid = (rowCount, colCount) => {
+  const gridNodes = [];
+  for (let row = 0; row < rowCount; row++) {
+    gridNodes.push([]);
+    for (let col = 0; col < colCount; col++) {
+      gridNodes[gridNodes.length - 1].push(new GridCell(true));
+    }
+  }
+  return gridNodes;
+};
```

Comment:
We could also add an optional bool parameter for highlight in the function above and utilize that function again.

Reply:
Depending on which PR get's merged first, this should also take in the fill color hex value. The GridCell now takes in the color of the cell instead of a boolean

Reply:
The handle select for fill all or fill unselected will need to have the same signature as the handle select method for `useActionSelectNode`, cause you will need to know what color code to use when coloring a given cell.